Library/Homebrew/rubocops/disable_comment.rb

```diff
@@ -0,0 +1,32 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Checks if rubocop disable comments have a clarifying comment preceding them.
+    class DisableComment < Base
+      MSG = "Add a clarifying comment to the RuboCop disable comment"
+
+      def on_new_investigation
+        super
+
+        processed_source.comments.each do |comment|
+          next unless disable_comment?(comment)
+          next if comment?(processed_source[comment.loc.line - 2])
```

Comment:
I noticed when I was fixing the offense for this cop in https://github.com/Homebrew/brew/pull/19084 that this passes if the comment line is only “#” with no words. Is that intentional?

Reply:
On it!

Reply:
Should be fixed and a I added a test for that case.